### PR TITLE
NMS-16025: modified tpl and adoc for issue

### PR DESCRIPTION
**Pull request by gallenc**
### All Contributors
additional property in core docker container to allow cassanrda setting of lab name
changes to tpl and adoc file to match.
See also issue in stack play https://github.com/opennms-forge/stack-play/issues/25

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-16025

* https://github.com/opennms-forge/stack-play/issues/25

### CODE REVIEW

**Review by indigo423 — DISMISSED** [review]
LGTM!

File: docs/modules/reference/pages/configuration/core-docker.adoc [hunk 1]
```diff
@@ -83,6 +83,7 @@
 | `OPENNMS_CASSANDRA_PORT`      | _Cassandra_ server port                                                           | optional | `9042`
 | `OPENNMS_CASSANDRA_USERNAME`  | Username with access to _Cassandra_                                               | optional | `cassandra`
 | `OPENNMS_CASSANDRA_PASSWORD`  | Password for user with access to _Cassandra_                                      | optional | `cassandra`
+| `OPENNMS_CASSANDRA_DATACENTER`  | data center DC set for  _Cassandra_                                             | optional | `datacenter1`
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/core-docker.adoc:86]:
```suggestion
| `OPENNMS_CASSANDRA_DATACENTER`  | Data center DC set for  _Cassandra_                                             | optional | `datacenter1`
```

**Review by Bonrob2 — COMMENTED** [review]
Minor update and question. Thanks!

**Review by indigo423 — APPROVED** [review]
LGTM!

**Review by Bonrob2 — APPROVED** [review]
LGTM! Thanks!

**Review by indigo423 — APPROVED** [review]
LGTM!

**Pull request merged**